components/sections/Send/index.tsx

```diff
@@ -75,10 +77,17 @@ function SendTokenRow({
 }
 
 export function Send(): ReactElement {
+	const {safeChainID} = useChainID();
 	const {configuration, dispatchConfiguration} = useSendFlow();
 	const {initialStateFromUrl} = useSendQueryManagement();
 
-	const {tokenList, getToken} = useTokenList();
+	const {tokenList} = useTokenList();
+
+	const initialTokensRaw =
+		initialStateFromUrl?.tokens?.map(token => ({address: toAddress(token), chainID: safeChainID})) || [];
+
+	const {data: initialTokens} = useBalances({tokens: initialTokensRaw});
```

Comment:
We should fetch the initial tokens + balance bc we are not sure that they are present in the token list